Scripts/developer_scripts/add_license_in_pkg_header.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Insert in all the header file of a package the include directive
+# "#include <CGAL/license/${package_name}.h" if it is not already there
+# There are two arguments to the script as some packages are split into
+# subdirectories (Polyhedron_IO, Algebraic_kernel_for_circles, ...)
+
+from sys import argv
+import os
+from os.path import join
+import codecs
+import re
+
+def add_license_include_in_file(package_name, fname):
+  # first, see if the include directive is already there
+  with codecs.open(fname, encoding='utf-8') as f:
```

Comment:
You are actually quite right. All our files are correctly encoded in ascii or utf-8, but one file:

- `Mesh_2/include/CGAL/Mesh_2/Mesh_global_optimizer_2.h` (coded with ISO-8859)

I have verified with:

    file ^build*/(doc|examples|test|demo|include)/**/*.(h*|cpp) | grep -vE '(UTF-8 Unicode|ASCII) text(, with very long lines)?$'

And you are also right that Git seems to translate the files end-of-lines, because all of them have Unix end-of-lines on my machine.